KubeArmor/core/dockerHandler.go

```diff
@@ -129,7 +129,7 @@ func (dh *DockerHandler) GetContainerInfo(containerID string) (tp.Container, err
 			container.EndPointName = val
 		}
 	}
-
+	container.ContainerImage, _ = K8s.GetContainerImage(container.NamespaceName, container.EndPointName, container.ContainerID)
```

Comment:
When KubeArmor started, it is possible that already deployed containers are first detected before k8s pods are detected. In that case, we may have unexpected errors.